js/src/dom/class-list.js

```diff
@@ -0,0 +1,27 @@
+function addRemoveClass(add, element, className, classNameList) {
+  if (!classNameList.length) {
+    element.classList[add ? 'add' : 'remove'](className)
+
+    return
+  }
+
+  const classList = [className].concat(classNameList)
+
+  classList.forEach(addClassName => element.classList[add ? 'add' : 'remove'](addClassName))
```

Comment:
could this be rewritten as

`classList.forEach(item => element.classList.toggle(item, add))` ?

Reply:
I prefer not to do that because if someone call `removeClass` on an element which doesn't possess the class it'll add it